modules/gdnative/gdnative_api.json

```diff
@@ -6449,6 +6449,12 @@
           "return_type": "bool",
           "arguments": [
           ]
+        },
```

Comment:
We should up the version of this API so we can check in the module whether we're on a new enough Godot build. Officially this should be a new API pointed to by NEXT in the old one but personally I think that is overkill when adding a single method.

Reply:
Done!